index.html

```diff
@@ -10,22 +10,44 @@
 </head>
 <body>
     <div class="body">
+        <nav class="top-nav">
+            <h2 >Awesome Books</h2>
+            <ul >
+                <li id="list"><a href="#">List</a></li>
+                <li id="add-new"><a href="#">Add new</a></li>
+                <li id="contact"><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+
         <header>
-            <h1>All Awesome books</h1>
+            <h1 id="awesome-heading">All Awesome books</h1>
+            <!-- <h1 id="contact-info">Contact Information</h1> -->
         </header>
 
         <section class="list-section">
             <ul id="book-list">
             </ul> 
         </section>
-        <hr />
 
-        <form>
+        <section class="contact-info-section">
+            <h1 >Contact Information</h1>
+            <p>Do you have any question or you just want to say "Hello"?</p>
+            <p>You can reach out to us!</p>
+            <ul class="contact-ul">
+                <li>Our e-mail: simpleshaikh2@gmail.com</li>
+                <li>Our phone number: 09840038288494983</li>
+                <li>Our address: Streetname 22, 84839 City, Country.</li>
+            </ul>
+        </section>
+
+        <form id="form" class="form1">
             <h3>Add a new book</h3>
             <p> <input type="text" placeholder="Title" id="title"> </p>
             <p> <input type="text" placeholder="Author" id="author"> </p>
             <button type="submit" id="add">Add</button>
         </form>
```

Comment:
- [ ] Ouch! this is really close but at a given moment only one of the sections should be displayed and you seem to be displaying all of the sections. check the video below 👇
    ![firefox_VMYJST7017](https://user-images.githubusercontent.com/104968236/236220425-09ad0a2a-616c-46c3-b06d-7c62a219fc70.gif)
- As you can see when the page loads all the sections are displayed so please work on that ✔
- **HINT**: You can give the "Contact" & "Add book" sections a default value of `display: none;`.